src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java

```diff
@@ -0,0 +1,114 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.oaipmh.service.ErrorsService;
+import org.folio.oaipmh.service.InstancesService;
+import org.folio.rest.jaxrs.resource.OaiPmhCleanUpErrorLogs;
+import org.folio.rest.jaxrs.resource.OaiPmhCleanUpInstances;
+import org.folio.s3.client.FolioS3Client;
+import org.folio.spring.SpringContextUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.ws.rs.core.Response;
+import java.io.File;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.folio.oaipmh.Constants.OKAPI_TENANT;
+import static org.folio.oaipmh.Constants.REPOSITORY_FETCHING_CLEAN_ERRORS_INTERVAL;
+import static org.folio.rest.jaxrs.resource.OaiPmhCleanUpInstances.PostOaiPmhCleanUpInstancesResponse.respond500WithTextPlain;
```

Comment:
Please move static imports to the top of the file.

Reply:
fixed